lib/main_common.pm

```diff
@@ -172,14 +173,17 @@ sub is_bridged_networking {
     if ($@) {
         bmwqemu::fctwarn("Failed to load main_ltp.pm:\n$@", 'main_common.pm');
         eval q%
-            sub maybe_load_kernel_tests {
-                if (get_var('INSTALL_LTP')
-                    || get_var('LTP_SETUP_NETWORKING')
-                    || get_var('LTP_COMMAND_FILE')
-                    || get_var('INSTALL_KOTD')
-                    || get_var('QA_TEST_KLP_REPO')
-                    || get_var('INSTALL_KOTD')
-                    || get_var('VIRTIO_CONSOLE_TEST'))
+            sub is_kernel_test {
```

Comment:
This could be moved into main_common as a normal function. It is a bit annoying to have it outside of main_ltp, but that is better than writting it twice.